### DISCUSSION TITLE
Change server brand

### DISCUSSION THREAD

**Discussion by NotThatRqd** [Minestom/Minestom#1924] [Help] [answered]
![image](https://github.com/Minestom/Minestom/assets/67846317/b898581c-196b-447d-b2e8-11423d5bcfa7)

How do I change this?

**Comment by GreatWyrm** [comment, marked as answer]
[MinecraftServer.setBrandName()](https://javadoc.minestom.net/net/minestom/server/MinecraftServer.html#setBrandName(java.lang.String))

**Reply by NotThatRqd** [reply]
thank you!
